_config.yml

```diff
@@ -24,6 +24,8 @@ nodecontainer: calico/node
 noderunning: calico-node
 
 open_source: true
+# Change this to v([0-9]+.[0-9])(?:.*?)$ for EE merge
```

Comment:
Why this comment? 

Also, why do we need to change it of the other docs site? Version scheme is more or less the same yeah?

Reply:
For EE docs website @bmckercher123 and @bcreane suggested a scheme like xx.y to be shown in the dropdown. In OSS this format leads to an incorrect visual representation of the version that is why I'm using a regex to seperate these two and EE pattern is different.

Reply:
@frozenprocess can you clean this up a bit to do the right thing for the calico/calico-private, e.g. https://github.com/projectcalico/calico/blob/master/_layouts/docwithnav.html#L183